camera_base.orogen

```diff
@@ -208,6 +208,10 @@ task_context "Preprocess", subclasses: "Base" do
     input_port('iframe', ro_ptr('base::samples::frame::Frame'))
     output_port 'oframe', ro_ptr('base::samples::frame::Frame')
 
+    property("undistort_algorithm","/frame_helper/InterpolationAlgorithm",:INTER_LINEAR).
+    doc "interpolation algorithm which is used to undistort the frame before it is written to the output port. "
+        "allowed values are INTER_LINEAR, INTER_NEAREST, INTER_AREA, INTER_CUBIC, INTER_LANCZOS4, BAYER_RESIZE"
```

Comment:
BAYER_RESIZE does not make sense for undistorting. And in the related PR https://github.com/rock-core/perception-frame_helper/pull/21 you reverted the renaming to `InterpolationAlgorithm`.

Reply:
Thank you